MapboxNavigation/CarPlayManager.swift

```diff
@@ -457,48 +435,20 @@ extension CarPlayManager: CPMapTemplateDelegate {
         let mapTemplate = CPMapTemplate()
         mapTemplate.mapDelegate = self
 
-        let showFeedbackButton = CPMapButton { [weak self] (button) in
-            self?.currentNavigator?.showFeedback()
-        }
-        showFeedbackButton.image = UIImage(named: "carplay_feedback", in: .mapboxNavigation, compatibleWith: nil)
-
-        let overviewButton = CPMapButton { [weak self] (button) in
-            guard let navigationViewController = self?.currentNavigator else {
-                return
-            }
-            navigationViewController.tracksUserCourse = !navigationViewController.tracksUserCourse
-
-            let imageName = navigationViewController.tracksUserCourse ? "carplay_overview" : "carplay_locate"
-            button.image = UIImage(named: imageName, in: .mapboxNavigation, compatibleWith: nil)
+        if let carPlayMapViewController = self.carWindow?.rootViewController as? CarPlayMapViewController,
+            let mapButtons = delegate?.carPlayManager?(self, mapButtonsCompatibleWith: carPlayMapViewController.traitCollection, in: mapTemplate, for: .navigating, carPlayMapViewController: carPlayMapViewController) {
+            mapTemplate.mapButtons = mapButtons
         }
-        overviewButton.image = UIImage(named: "carplay_overview", in: .mapboxNavigation, compatibleWith: nil)
-
-        mapTemplate.mapButtons = [overviewButton, showFeedbackButton]
 
         if let rootViewController = self.carWindow?.rootViewController as? CarPlayMapViewController,
             let leadingButtons = delegate?.carPlayManager?(self, leadingNavigationBarButtonsCompatibleWith: rootViewController.traitCollection, in: mapTemplate, for: .navigating) {
             mapTemplate.leadingNavigationBarButtons = leadingButtons
         }
         
-        let muteTitle = NSLocalizedString("CARPLAY_MUTE", bundle: .mapboxNavigation, value: "Mute", comment: "Title for mute button")
-        let unmuteTitle = NSLocalizedString("CARPLAY_UNMUTE", bundle: .mapboxNavigation, value: "Unmute", comment: "Title for unmute button")
-        
-        let muteButton = CPBarButton(type: .text) { (button: CPBarButton) in
```

Comment:
The mute/unmute button is useful out of the box; we don’t want to force developers to copy-pasta swaths of the example application to get such basic functionality. Let’s add factory methods for creating these buttons, which will result in restoring the localized strings.